tools/python_api/src_cpp/pandas/pandas_scan.cpp

```diff
@@ -152,7 +152,7 @@ static bool isPyArrowBacked(const py::handle& df) {
     return false;
 }
 
-static std::unique_ptr<ScanReplacementData> tryReplacePD(py::dict& dict, py::str& objectName) {
+std::unique_ptr<ScanReplacementData> tryReplacePD(py::dict& dict, py::str& objectName) {
```

Comment:
why remove `static` now?

Reply:
I've replaced replacePD with replacePythonObject to reuse the frame lineage search code. As a result, tryReplacePD had to be included in the header file.